db-lb-app/infrastructure/openstack.yaml

```diff
@@ -8,24 +8,33 @@ imports:
 
 inputs:
 
-  region_name:
-    type: string
+  openstack_region:
+    description: ""
+    type:
+    default: { get_secret: openstack_region }
 
   public_subnet_cidr:
+    description: ""
```

Comment:
Why do you need empty descriptions? Remove them if they aren't going to say anything. They are not required.

Reply:
added description content